File: AUtil/app/src/main/java/com/example/administrator/base/Application.java
package com.example.administrator.base;

import com.facebook.stetho.Stetho;

/**
 * Created by 舍长 on 2018/4/27.
 */

public class Application extends android.app.Application{
    @Override
    public void onCreate() {
        super.onCreate();
        // 数据库文件查看框架
        Stetho.initializeWithDefaults(this);
    }
}

File: BaseProject/app/src/main/java/com/example/administrator/app/app/Application.kt
package com.example.administrator.app.app

import android.app.Application
import com.facebook.stetho.Stetho

/**
 * Created by 舍长 on 2018/6/9.
 * 舍长:
 */
class App : Application() {
    override fun onCreate() {
        super.onCreate()
        Stetho.initializeWithDefaults(this)
    }
}